modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFolderItemLocalServiceTest.java

```diff
@@ -264,6 +279,51 @@ public void testDeleteObjectFolderItemByObjectFolderId() {
 						_objectFolderA.getObjectFolderId())));
 	}
 
+	@Test
+	public void testGetObjectFolderItemByObjectFolderId() throws Exception {
+		String originalName = PrincipalThreadLocal.getName();
+		PermissionChecker originalPermissionChecker =
+			PermissionThreadLocal.getPermissionChecker();
+
+		try {
+			User user = UserTestUtil.addUser();
+
+			PermissionThreadLocal.setPermissionChecker(
+				PermissionCheckerFactoryUtil.create(user));
+			PrincipalThreadLocal.setName(user.getUserId());
+
+			Role role = RoleTestUtil.addRole(RoleConstants.TYPE_REGULAR);
+
+			_resourcePermissionLocalService.setResourcePermissions(
+				TestPropsValues.getCompanyId(), ObjectFolder.class.getName(),
+				ResourceConstants.SCOPE_COMPANY,
+				String.valueOf(TestPropsValues.getCompanyId()),
+				role.getRoleId(), new String[] {ActionKeys.VIEW});
+
+			_userLocalService.addRoleUser(role.getRoleId(), user);
+
+			ObjectFolder objectFolder =
+				_objectFolderLocalService.getUncategorizedObjectFolder(
+					TestPropsValues.getCompanyId());
+
+			List<ObjectFolderItem> objectFolderItems =
+				_objectFolderItemLocalService.
+					getObjectFolderItemsByObjectFolderId(
+						objectFolder.getObjectFolderId());
+
+			Assert.assertEquals(
+				objectFolderItems.toString(), 0, objectFolderItems.size());
+
```

Comment:
Maybe you can add a new scenario in this test:
Once this ObjectFolder has some ObjectDefinition linked. Give view permission over this ObjectDefinition and assert that the user can _see_ it